website/templates/emails/reviews_submission_status.txt.mako

```diff
@@ -1,9 +1,9 @@
 Hello ${user.fullname},
```

Comment:
Can we safely remove the `txt` templates? They don't seem to be used and it's annoying to keep the two in sync.

Reply:
I was hesitant before but the B&I team is working to get rid of all of the txt templates so I think removing them is best 👍